index.html

```diff
@@ -213,6 +213,29 @@ <h2>
             YES
           </td>
         </tr>
+        <tr data-cite="notifications">
+          <td class="string-token">
+            "<a><code>notifications</code></a>"
```

Comment:
(one issue for discussion, perhaps, we made `<code>` part of the dfn for notifications per the editor's request. do we want to do that for all powerful features?)